client/public/clientJS/adminWishCards.js

```diff
@@ -7,13 +7,19 @@ window.onload = function () {
 
 function unlockCardButtonClick(e) {
     const cardId = e.target.dataset.valueId;
+    const oldItemUrl = document.getElementById("oldWishItemUrl"+cardId);
```

Comment:
since we have jquery basically everywhere it might make sense to use it here too